### PR TITLE
rabbit-task example

### PR DESCRIPTION
**Pull request by meirshal**

### CODE REVIEW

File: examples/calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+var config = new Configuration(conn);
+var worker = new Worker(
+  'calculator-worker-worker',
+  conn,
+  queueName,
+  router,
+  function() { return config.configure() }
+);
+
+worker.start();
```
Comment (asafdav) [inline examples/calculator/README.md]: Awesome !

File: examples/calculator/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/**
+ * Created by meirshalev on 31/01/2017.
+ */
+
+console.log('Hello, World!');
```
Comment (asafdav) [inline examples/calculator/index.js]: what's that ?

File: examples/calculator/worker/calculatorHandler.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by meirshalev on 05/02/2017.
+ */
+var schema = require('./schema');
+var ContextStoreHandler = require('../../../lib/handlers/contextStoreHandler');
+var math = require('mathjs'); // A nice math library that can evaluate string expressions.
+var constants = require('../constants');
+
+calculatorHandler =  {
```
Comment (asafdav) [inline examples/calculator/worker/calculatorHandler.js]: you should define your variables before you use them (strict mode)

File: examples/calculator/worker/calculatorHandler.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  name: 'calculatorHandler',
+  pattern: constants.expressionEvalTopic,
+  base: ContextStoreHandler,
```
Comment (asafdav) [inline examples/calculator/worker/calculatorHandler.js]: why do you need the contextStoreHandler ?

File: examples/calculator/worker/calculatorHandler.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  }
+};
+
+function evaluate(msg) {
+  var expression = msg.content.expression;
+  console.log('received expression: ' + expression);
+
+  var result = math.eval(expression);
+  console.log('result: ' + result);
+  return { result: result };
```
Comment (asafdav) [inline examples/calculator/worker/calculatorHandler.js]: might be better to return Promise.resolve(result) so people know they can use promises

File: examples/calculator/index.js (hunk 2 above)
Comment (meirshal) [inline examples/calculator/index.js]: I forgot to delete this file. Only used it to see that I my gulp tasks work.
Now deleted.

File: examples/calculator/worker/calculatorHandler.js (hunk 3 above)
Comment (meirshal) [inline examples/calculator/worker/calculatorHandler.js]: Done.

File: examples/calculator/worker/calculatorHandler.js (hunk 5 above)
Comment (meirshal) [inline examples/calculator/worker/calculatorHandler.js]: Done.

File: examples/calculator/worker/calculatorHandler.js (hunk 4 above)
Comment (meirshal) [inline examples/calculator/worker/calculatorHandler.js]: I don't. Replaced it with JsonSchemaHandler.

**Pull request merged**